# authentic2 idp saml french l10n
# Copyright (C) 2014 Entr'ouvert
# This file is distributed under the same license as the authentic package.
# Benjamin Dauvergne <bdauvergne@entrouvert.com>, 2014.
#
msgid ""
msgstr ""
"Project-Id-Version: Authentic\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-01-27 14:19+0100\n"
"PO-Revision-Date: 2015-12-03 21:34+0100\n"
"Last-Translator: Mikaël Ates <mates@entrouvert.com>\n"
"Language-Team: None\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n>1;\n"

#: src/authentic2/idp/saml/__init__.py:28
msgid "SAML2"
msgstr ""

#: src/authentic2/idp/saml/backend.py:136
msgid "Delete"
msgstr "Supprimer"

#: src/authentic2/idp/saml/backend.py:151
msgid "Create"
msgstr "Créer"

#: src/authentic2/idp/saml/saml2_endpoints.py:453
#, python-format
msgid ""
"SAMLv2 Single Sign On: invalid message for WebSSO profile with HTTP-Redirect "
"binding: %r"
msgstr ""
"SAMLv2 Single Sign On: message invalide pour le profil utilisant le binding "
"HTTP-Redirect: %r"

#: src/authentic2/idp/saml/saml2_endpoints.py:458
msgid ""
"SAMLv2 Single Sign On: the request cannot be answered because no valid "
"protocol binding could be found"
msgstr ""
"SAMLv2 Single Sign On: la requête ne peut être traitée car elle est "
"invalide: un binding de protocole n'a pu être trouvé"

#: src/authentic2/idp/saml/saml2_endpoints.py:493
msgid "sso: No SP policy defined"
msgstr "sso: Pas de politique pour les fournisseurs de service définie"

#: src/authentic2/idp/saml/saml2_endpoints.py:613
msgid "continue_sso: error loading login"
msgstr "continnue_sso: erreur au rechargement de l'objet login"

#: src/authentic2/idp/saml/saml2_endpoints.py:617
#, python-format
msgid "continue_sso: unknown provider %s"
msgstr "continue_sso: fournisseur %s inconnu"

#: src/authentic2/idp/saml/saml2_endpoints.py:716
#, python-format
msgid "Provider %s is unknown"
msgstr "Le fournisseur %s est inconnu"

#: src/authentic2/idp/saml/saml2_endpoints.py:720
msgid "No service provider policy defined"
msgstr "Pas de politique de fournisseur de service définie"

#: src/authentic2/idp/saml/saml2_endpoints.py:869
#: src/authentic2/idp/saml/saml2_endpoints.py:1056
#: src/authentic2/idp/saml/saml2_endpoints.py:1069
#: src/authentic2/idp/saml/saml2_endpoints.py:1302
#: src/authentic2/idp/saml/saml2_endpoints.py:1336
#: src/authentic2/idp/saml/saml2_endpoints.py:1385
#: src/authentic2/idp/saml/saml2_endpoints.py:1396
#, python-format
msgid "You are being redirected to \"%s\""
msgstr "Vous allez être redirigé sur \"%s\""

#: src/authentic2/idp/saml/saml2_endpoints.py:1388
msgid "Invalid logout request"
msgstr "Requête de déconnexion invalide"

#: src/authentic2/idp/saml/templates/idp/saml/logout_fragment.html:2
#, python-format
msgid "Sending logout to %(name)s..."
msgstr "Déconnexion avec %(name)s..."

#: src/authentic2/idp/saml/templates/idp/saml/unknown_provider.html:5
#, python-format
msgid ""
"We were unable to load provider\n"
"  <tt>%(entity_id)s</tt>. \n"
"  You can <a href=\"%(add_url)s\">add</a> it now or <a href=\"\">retry</a>.\n"
"   "
msgstr ""
"Nous n'avons pas pu charger le fournisseur de service\n"
"  <tt>%(entity_id)s</tt>. \n"
"  Vous pouvez <a href=\"%(add_url)s\">l'ajouter</a> maintenant ou <a href="
"\"\">réessayer</a>.\n"
"   "

#: src/authentic2/idp/saml/views.py:26
#, python-brace-format
msgid "Federation to {0} deleted"
msgstr "La liaison avec {0} a été supprimée."
